content/en/docs/components/katib/experiment.md

```diff
@@ -754,11 +754,7 @@ kubectl apply -f <your-path/your-experiment-config.yaml>
 
 - (Optional) Katib's experiments don't work with
   [Istio sidecar injection](https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection).
-  If you install Kubeflow using
-  [Istio config](https://www.kubeflow.org/docs/methods/kfctl/deployment),
-  you have to disable sidecar injection. To do that, specify this annotation:
-  `sidecar.istio.io/inject: "false"` in your experiment's trial template. For
-  examples on how to do it for `Job`, `TFJob` (TensorFlow) or
```

Comment:
We should not delete this information since the default installation is also used Istio: https://github.com/kubeflow/manifests/blob/master/example/kustomization.yaml#L9.
Maybe we can just rephrase it ? For instance: 
"If you are running Kubeflow with Istio config, you have to disable sidecar injection."